chocolatey/notable.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>notable</id>
+    <version>1.3.0</version>
+    <packageSourceUrl>https://github.com/fabiospampinato/notable</packageSourceUrl>
+    <owners>Fabio Spampinato</owners>
+    <title>notable (Install)</title>
+    <authors>Fabio Spampinato</authors>
+    <projectUrl>https://github.com/fabiospampinato/notable</projectUrl>
+    <iconUrl>https://github.com/fabiospampinato/notable/blob/master/resources/icon/icon.png?raw=true</iconUrl>
+    <copyright>2018-present Fabio Spampinato</copyright>
+    <licenseUrl>https://raw.githubusercontent.com/fabiospampinato/notable/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/fabiospampinato/notable</projectSourceUrl>
+    <docsUrl>https://github.com/fabiospampinato/notable/tree/master/resources/tutorial/notes</docsUrl>
+    <bugTrackerUrl>https://github.com/fabiospampinato/notable/issues</bugTrackerUrl>
+    <tags>notable md markdown notes note-taking code text editor</tags>
+    <summary>The markdown-based note-taking app that doesn't suck.</summary>
+    <description> Notable is a:
```

Comment:
This doesn't read very well. Could it be re-written to something like:

> Notable is a free, open source note taking app with the following philosophy

Could use ideals instead of philosophy, if that sounds better...

Reply:
It could be, I was just copying the readme. I figured it was the safe choice at the time.

Reply:
I might put in a PR to improve the readme then :)